{{cookiecutter.github_repo_name}}/tools/db-safe-for-dev.sh

```diff
@@ -0,0 +1,23 @@
+# Change database UUID
+echo "Change database UUID"
+new_uuid=$(uuidgen)
+docker exec {{cookiecutter.project_code}}_postgres psql -U odoo -c "UPDATE ir_config_parameter SET value = '${new_uuid}' WHERE key='database.uuid';" "$@"
+echo "New UUID: ${new_uuid}"
+
+# Disable Mail Server
+echo "Disabling mail server"
+docker exec {{cookiecutter.project_code}}_postgres psql -U odoo -c "UPDATE ir_mail_server SET active = false;" "$@"
+docker exec {{cookiecutter.project_code}}_postgres psql -U odoo -c "UPDATE ir_mail_server SET smtp_user = CONCAT(smtp_user, '.test'), smtp_host = CONCAT(smtp_host, '.test');" "$@"
+
+# Disable Cron
+echo "Disabling cron"
+docker exec {{cookiecutter.project_code}}_postgres psql -U odoo -c "UPDATE ir_cron SET active = false;" "$@"
+
+# Change admin password
+echo "Change admin password"
+python_code="from passlib.context import CryptContext
+setpw = CryptContext(schemes=['pbkdf2_sha512'])
+print(setpw.hash('admin'))"
+admin_password=$(python3 -c "$python_code")
+echo "New password: ${admin_password}"
+docker exec {{cookiecutter.project_code}}_postgres psql -U odoo -c "UPDATE res_users SET login='admin', password = '${admin_password}' WHERE id=2;" "$@"
```

Comment:
@OpenCode Come la vedi togliere la WHERE? In modo da resettarle tutte e amen.

Reply:
Dovrei fare due query diverse perché qui setto anche l'username di accesso. Ma posso farlo senza problemi magari impostando le altre a "test"

Reply:
Vero, avevo perso SET login='admin'. Guarda, considera pure che in qualche raro caso admin non è id 2.
My take: considerato che:
1- è raro che admin non si chiami admin
2- spesso capita di dover entrare con un altro account
3- potrebbe non essere id=2 e perderci pezzi
Io farei banalmente un UPDATE per SETtare admin come password dappertutto, e basta. Che ne dici?

Reply:
Ho fatto la modifica per settare le password anche degli utenti. Io proporrei di inizare così. Quello dell'admin con id != 2 è un caso talmente limite che non mi preoccuperei per ora perché al massimo lo si gestisce manualmente e poi, studiando il caso, si può trovare una soluzione più fine. Le query, per adesso, sono pure abbastanza rozze e il codice bash non è bellissimo. Ma meglio di niente...